### PR TITLE
Cleanup bootc related info from repo

### PR DESCRIPTION
**Pull request by oglok**
Moved to flightctl-demos repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,78 +73,3 @@ podman-compose -f deploy/podman/observability.yaml up
 ```
 
 The Grafana and Prometheus web UIs are then accessible on `http://localhost:3000` and `http://localhost:9090`, respectively.
-
-
-## Agent Image Creation
```
[the remark this answers is not in the document]
Comment (fzdarsky) [reply README.md:78]: We're moving this to the `flightctl-demos` repo that'll contain examples and use cases.
Right now, the section is a bit outdated, written for developers (assumes local RPM build), has artefacts in `packaging` (though we don't expect to provide OOTB images), and is specific to bootc (though we want to support rpm-ostre, too).

[the remark this answers is not in the document]
Comment (hexfusion) [reply README.md:78]: just my 2c but I think we would be better off just putting everything in a single repo. then just gate it on OWNERS file if we have specific users who can merge directly. This way the chances on the examples being tested and maintained are much higher. 
 
IE /examples/

[the remark this answers is not in the document]
Comment (fzdarsky) [reply README.md:78]: Examples being tested and maintained is a good point. We were thinking a dedicated repo would have the advantage that we can directly RepoSync into the FlightCtl service and users can fork without the baggage of the FlightCtl code base. And then potentially also set up a production-like GitOps experience (with gating tests) there.

[the remark this answers is not in the document]
Comment (oglok) [reply README.md:78]: should we then approve this? wdyt?

**Pull request merged**